test_create_garden.py

```diff
@@ -0,0 +1,113 @@
+import unittest
+from main import create_garden, counting_items, plant, day_increment, grow_garden
```

Comment:
Why are we not testing grow_garden, plant_randomly, display_garden?

Reply:
i did try to test grow garden , but for it to work and test , i need the day increment function to work first . 
how do i test the 10% chance of a flower or weed forming  in plant randomly ?

Reply:
Isn't there any way we can test it?